File: code_wars/kata/anagram.rb
def anagrams(word, words)
  ans = []
  for x in words do
    if word.chars.sort_by(&:downcase).join == x.chars.sort_by(&:downcase).join
      ans.push(x)
    end
  end
  print ans
  return ans
end


anagrams("abba",["aabb", "abcd", "bbaa", "dada"])

File: code_wars/kata/anagram/anagram.py
def anagrams(word, words):
    gold = word
    ans = []
    for x in words:
        if sorted(x) == sorted(gold):
            ans.append(x)
    print(ans)

anagrams('abba', ['aabb', 'abcd', 'bbaa', 'dada'])
